test/test-tools.go

```diff
@@ -43,14 +43,14 @@ func AssertNotNil(t *testing.T, obj interface{}, message string) {
 // AssertNotError checks that err is nil
 func AssertNotError(t *testing.T, err error, message string) {
 	if err != nil {
-		t.Fatalf("%s %s", caller(), message)
+		t.Fatalf("%s %s: %s", caller(), message, err)
```

Comment:
Good point. I was duplicating what was there before my change, but this is better. Fixed.